ga4gh/datamodel/datasets.py

```diff
@@ -227,6 +226,27 @@ def __init__(self, localId, dataDir, backend):
                     self, localId, bamPath, backend)
                 self.addReadGroupSet(readGroupSet)
 
+    def descriptionExists(self, fname):
+        # check for existence of description JSON
+        return os.path.exists(fname)
+
+    def parseDescriptionJson(self, str):
+        # parse into python dict
+        return json.loads(str)
+
+    def descriptionFilename(self, path, localId):
+        return os.path.join(path, localId + ".json")
+
+    def isAnnotated(self, path, localId):
+        ret = False
+        fname = self.descriptionFilename(path, localId)
+        if self.descriptionExists(fname):
```

Comment:
I'm not sure the ``descriptionExists`` does much for readablility here.How about
```
fname = self.getDescriptionFilename(path, localId)
if os.path.exists(fname):
    # etc.
```